index.bs

```diff
@@ -623,3 +637,44 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
     </p>
   </section>
 </section>
+
+
+# Examples # {#examples}
+
+## Query recording capabilities with {{encodingInfo()}} ## {#example1}
+
+<div class="note">
+  The following example can also be found in e.g. <a href="https://codepen.io/miguelao/pen/bWNwej/left?editors=0010#0">this codepen</a> with minimal modifications.
```

Comment:
Interesting. I've never seen a spec pointing to real code but I guess it's quite useful :) 

Can you line-wrap to 80 characters?